api/proto/common.proto

```diff
@@ -0,0 +1,72 @@
+// Copyright 2025 The Sigstore Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+package dev.sigstore.rekor.v2;
+
+import "google/api/field_behavior.proto";
+
+option go_package = "github.com/sigstore/rekor-tiles/pkg/generated/protobuf";
+
+// A signature over some content
+message Signature {
+    // The signature over the content
+    bytes content = 1 [(google.api.field_behavior) = REQUIRED];
+    // The key and scheme used to verify the signature
+    Verifier verifier = 2 [(google.api.field_behavior) = REQUIRED];
```

Comment:
Thoughts on removing Verifier from a Signature, and having a HashedRekord type be a Signature, Verifier, and Hash?

Reply:
done